ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -228,17 +228,27 @@ public void UpdateTextBlock(TextBlock textBlock)
 
         private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
         {
-            //textBlock.CharacterSpacing = _letterSpacing;
-            //textBlock.MaxLines = _numberOfLines;
-            textBlock.LineHeight = _lineHeight != 0 ? _lineHeight : double.NaN;
+            textBlock.LineHeight = Math.Abs(_lineHeight) > 0.1 ? _lineHeight : double.NaN;
             textBlock.TextAlignment = _textAlignment;
-            textBlock.FontFamily = _fontFamily != null ? new FontFamily(_fontFamily) : new FontFamily();
-            textBlock.FontSize = _fontSize ?? 15;
-            textBlock.FontStyle = _fontStyle ?? new FontStyle();
+
+            if (_fontFamily != null)
+            {
+                textBlock.FontFamily = new FontFamily(_fontFamily);
+            }
+
+            if (_fontSize.HasValue)
+            {
+                textBlock.FontSize = _fontSize.Value;
```

Comment:
If the "fontSize" property was not specified we were setting it to 15, which was different than the default for WPF. This change ensures that we wouldn't override the system default of the property was not set..